src/commands/clone-sandbox/index.js

```diff
@@ -1,4 +1,4 @@
-const { getSandboxFiles, createSandboxFiles } = require('./utils');
+const { getSandboxFiles, createSandboxFiles } = require('../../utils/get-sandbox');
```

Comment:
I would suggest keeping all the util method in `index.js` file itself rather than creating a new file in utils folder